apps/web/src/pages/templates/components/ListProviders.tsx

```diff
@@ -79,20 +85,58 @@ export const ListProviders = ({
                 setConfigureChannel(provider.channel);
               }}
             >
-              <Group position="apart">
-                <img
-                  src={'/static/images/providers/' + colorScheme + '/' + provider.logoFileName[`${colorScheme}`]}
-                  alt={provider.displayName}
-                  style={{
-                    height: '24px',
-                    maxWidth: '140px',
-                    opacity: provider.active ? 1 : colorScheme === 'dark' ? 0.4 : 1,
-                  }}
-                />
-                <Text color={provider.active ? colors.success : colors.B60}>
-                  <When truthy={provider.active}>Active</When>
-                  <When truthy={!provider.active}>Disabled</When>
-                </Text>
+              <Group spacing={16} position="apart">
+                <Group spacing={16} position="apart">
+                  <img
+                    src={'/static/images/providers/' + colorScheme + '/square/' + provider.providerId + '.svg'}
+                    alt={provider.displayName}
+                    style={{
+                      height: '24px',
+                      maxWidth: '140px',
+                      opacity: provider.active ? 1 : colorScheme === 'dark' ? 0.4 : 1,
+                    }}
+                  />
+
+                  <Stack
+                    sx={{
+                      width: isIntegrationsListPageEnabled ? '117px' : undefined,
+                    }}
+                    spacing={0}
+                  >
+                    <Tooltip label={provider.displayName} opened={provider.name ? undefined : false}>
```

Comment:
```suggestion
                    <Tooltip label={provider.displayName} opened={isIntegrationsListPageEnabled}>
```
I think after the migration, all will have a name, also those that don't have the feature. You could also use the hidden property hidden={!isIntegrationsListPageEnabled} instead of open, to avoid the undefined